mkdocs/themes/mkdocs/css/base.css

```diff
@@ -226,24 +240,110 @@ blockquote {
     text-align: left;
```

Comment:
(detached comment)

Hmm headerlinks look quite ugly - the chain logo is always underlined now, so it's hard to see that it's even supposed to be a chain. 

Also it's more heavily misaligned from center.

How about this addition:

```
.headerlink {
    text-decoration: none;
    vertical-align: middle;
}
```

Reply:
Applied this suggestion in https://github.com/mkdocs/mkdocs/commit/351b87939ddc61bc767e2b869e2a2673fe9af5ae